src/aat/resources/json/create_online_hearing.json

```diff
@@ -1,3 +1 @@
-{
-  "externalRef" : "case_id_123"
-}
\ No newline at end of file
+{"externalRef" : "case_id_97", "jurisdictionName" : "SSCS"}
```

Comment:
FYI the API is now up to 0.0.4 and request payload has now changed (again). We'll create another story to make the payload look lie the API spec.